src/store/BouquetStore.js

```diff
@@ -0,0 +1,56 @@
+import { defineStore } from "pinia"
+import TopicsAPI from "../services/api/resources/TopicsAPI"
+
+const topicsAPI = new TopicsAPI()
+
+export const useBouquetStore = defineStore("bouquet", {
+  state: () => ({
+    data: [],
+  }),
+  actions: {
+    /**
+     * Filter a list of bouquets related to Ecospheres
+     *
+     * @param {Array} bouquets
+     * @returns {Array}
+     */
+    filter (bouquets) {
+      return bouquets.filter(bouquet => bouquet.extras.is_ecospheres || bouquet.tags.includes("ecospheres"))
```

Comment:
We probably don't want to display the main "ecospheres" bouquet, especially when it'll contain thousands of datasets...

Reply:
True, I guess we can put the Ecospheres' bouquet id in the config file when it exists, and filter it out. Or add an extra `is_universe` or something.